app/src/androidTest/java/com/macbitsgoa/ard/activities/AuthActivityTest.java

```diff
@@ -0,0 +1,57 @@
+package com.macbitsgoa.ard.activities;
+
+import android.app.Activity;
+import android.app.Instrumentation;
+import android.support.test.espresso.intent.rule.IntentsTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import com.google.android.gms.common.ConnectionResult;
+import com.macbitsgoa.ard.R;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.intent.Intents.intended;
+import static android.support.test.espresso.intent.Intents.intending;
+import static android.support.test.espresso.intent.Intents.times;
+import static android.support.test.espresso.intent.matcher.IntentMatchers.hasComponent;
+import static android.support.test.espresso.matcher.ViewMatchers.isClickable;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static junit.framework.Assert.assertFalse;
+
+/**
+ * UI tests for AuthActivity.
+ * @author Rushikesh Jogdand
+ */
+@RunWith(AndroidJUnit4.class)
+public class AuthActivityTest {
+
+    @Rule
+    public IntentsTestRule<AuthActivity> activityTestRule =
+            new IntentsTestRule<>(AuthActivity.class);
+
+    @Test
+    public void testGoogleSignInExists() {
```

Comment:
Though technically not wrong, as we are clicking on this same button in the following test, is there any specific requirement for this one? 
Only reason for saying so is it is adding to the testing time unnecessarily though it is completely valid to have it.
I think we can avoid this test for the PR